table_version-loader.bash

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+set -o errexit -o noclobber -o nounset -o pipefail
```

Comment:
`nounset` is causing line 18 to fail, because the loop inevitably `shift`s past the entire array. The easy way to fix this is to use `"${1-}"

Reply:
is the "`" important here?

Reply:
Oh, whoops, that was meant to be Markdown formatting. Fixed.